tensornetwork/visualization/graphviz.py

```diff
@@ -14,13 +14,14 @@
 """Implementation of TensorNetwork Graphviz visualization."""
 
 import graphviz
-from typing import Optional, Text
+from typing import Optional, Text, Sequence
 from tensornetwork import network
 
 
 #pylint: disable=no-member
-def to_graphviz(net: network.TensorNetwork,
+def to_graphviz(net: network.TensorNetwork = None,
                 graph: Optional[graphviz.Graph] = None,
+                nodes: Optional[Sequence] = None,
```

Comment:
I also don't like having separate `net` and `nodes` arguments. Can you modify this so that either can be passed to the first argument?

Reply:
Also by merging the args into one, it gets rid of the need for the `None` checks.

Reply:
Sure, can I just Assert for the datatype and proceed?